docs/conferences/people.md

```diff
@@ -0,0 +1,36 @@
+In some parts of the event you might need to link other users.
+These could be authors and/or speakers in contributions and abstracts, event chairpersons, session conveners or
+lecture speakers.
+
+![](../assets/conferences/people/person-link.png)
+
+To add an existing user, use the search functionality by clicking on _Search_.
+You can search users by their name, email and affiliation:
+
+![](../assets/conferences/people/search.png)
+
+When a user is linked to an event, Indico stores a copy of the user's personal data in the event.
```

Comment:
Maybe "user's [details|data|information] like name and affiliation" is better than "personal data"?
Nowadays some people react a bit sensitive to naming things "personal data"...